skt/executable.py

```diff
@@ -179,7 +179,7 @@ def cmd_merge(cfg):
             for mb in cfg.get('merge_ref'):
                 save_state(cfg, {'mergerepo_%02d' % idx: mb[0],
                                  'mergehead_%02d' % idx: bhead})
-                (retcode, _) = ktree.merge_git_ref(*mb)
+                (retcode, bhead) = ktree.merge_git_ref(*mb)
```

Comment:
That's not `skt merge` but `git merge`. Same as we allow to apply multiple patches at the same time, we allow multiple git merges into the tree. To be fair I'm not sure if the option was ever used but since it's there, it should at least be fixed.

Reply:
Hehe.  Fair enough.